formulae/viewer/routes.py

```diff
@@ -0,0 +1,205 @@
+                                                             ,objectId=objectId, codex=codex, passage_data=passage_data, url=dict())
+            else:
+                manifest = url_for('viewer.static', filename=formulae["manifest"])
+                with open((current_app.IIIFmapping+"/"+formulae["manifest"]), "r") as f:
+                    title = load(f)
+                codex=title["label"]
+                return current_app.config['nemo_app'].render(template='viewer::multiviewermirador.html', manifest=manifest
+                                                             ,objectId=objectId, codex=codex, passage_data=passage_data, url=dict())
+        else:
+            try:
+                view = int(view)
+            except:
+                #exception when in the url view contains another character than number
+                view=0
+                images = formulae["images"]
+                folios = formulae["folios"]
+                town = formulae['town']
+                codex = formulae['codex']
+                current_folios = folios[view]
+                flash( _('There is not an images for this formula. Showing the first page.'))
+                link_picture=current_app.IIIFserver +str(images[view])
+                return current_app.config['nemo_app'].render(template='viewer::multiviewer.html', picture=link_picture, objectId=objectId,
+                                                         current_view=view, total_views=len(images), passage_data=passage_data,
+                                                         url=dict(), town=town, codex=codex, current_folios=current_folios,folios=folios)
+            images = formulae['images']
+            folios = formulae['folios']
+            town = formulae['town']
+            codex = formulae['codex']
+
+            #avoid error when the number of view is over the maximum
+            if view > len(images)-1:
+
+                flash( _('There are not {} images for this formula. Showing the last page image instead.'.format(view)))
+                view = len(images)-1
+                if "/images/formulae-1" in str(images[view]):
+                    link_picture=current_app.IIIFserver +str(images[view])
+                else:
+                    link_picture=str(images[view])
+                return current_app.config['nemo_app'].render(template='viewer::multiviewer.html', picture=link_picture, objectId=objectId,
+                                                         current_view=view, total_views=len(images), passage_data=passage_data,
+                                                         url=dict(), town=town, codex=codex, folios=folios)
+
+            #avoid error when the number of view is under the mainimum
+            elif view<0:
+                view=0
+                flash( _('There are not {} images for this formula. Showing the first page image instead.'.format(view)))
```

Comment:
Original messages should be in German. Plus this message doesn't make much sense since they are asking for a view that is less than 0. I would suggest the following message instead:

`flash(_('Die Zählung der Bilder fängt immer mit 0. Das erste Bild wird hier gezeigt.')`